deployment.yaml

```diff
@@ -83,8 +83,8 @@ objects:
         partitions: 4
         topicName: platform.payload-status
       - replicas: 3
-        partitions: 3
-        topicName: platform.upload.malware-detection
+        partitions: 64
```

Comment:
@r14chandra @SteveHNH not sure but, just wanted to know that is there any specific reason to set number of partitions to 64?

Reply:
@apuntamb I've referred [https://github.com/RedHatInsights/platform-mq/blob/master/topics/topics.json](Platform-mq)